client/webserver/locales/de-de.go

```diff
@@ -135,7 +135,7 @@ var DeDE = map[string]*intl.Translation{
 	"Export Account":            {T: "Account exportieren"},
 	"simultaneous_servers_msg":  {T: "Der <span class=brand></span> unterstützt die gleichzeitige Nutzung einer beliebigen Anzahl von DEX-Servern."},
 	"Change App Password":       {T: "App-Passwort ändern"},
-	"Build ID":                  {T: "Build ID"},
+	"Build ID":                  {T: "Version"},
```

Comment:
Should just add a translation for Version and update the UI.

Reply:
Thanks, again I will change this also.